lib/mkl/libonemkl.jl

```diff
@@ -42,6 +43,23 @@ function onemklZgemm(device_queue, transA, transB, m, n, k, alpha, A, lda, B, ld
                                     C::ZePtr{ComplexF64}, ldc::Int64)::Cint
 end
 
+function onemklDnrm2(device_queue, ctx, dev, n, x, incx, result)
+	@ccall liboneapi_support.onemklDnrm2(device_queue::syclQueue_t, ctx::syclContext_t, dev::syclDevice_t, n::Int64, x::ZePtr{Cdouble}, incx::Int64, result::RefOrZeRef{Cdouble})::Cvoid
```

Comment:
Better to maintain alignments.

Reply:
@Sarbojit2019  Thanks for review, yes I have manually corrected. @maleadt please share auto coding style format correction script in case if you maintain or please share the recommended practice on the same.